### ISSUE TITLE
[BUG] Terrorist Mute continues even if he moves to CT in the next round

### ISSUE THREAD

**Issue by paulocrash** [shanapu/MyJailbreak#315]
I believe I found a BUG in the Plugin...

In case you have the option to mutate the Terrorists a few seconds when the round begins, so the CT/Warden can order.

There is the problem, in case the Terrorist decides to change teams, he can not even speak in the MIC, he is mutated, only after the Terrorist mute ends, it happens even if he dies of Terrorist and passes to CT in the next round, can only speak after the mute is removed.

**MyJailBreak Version:** MyJB-master-58f4b42

**SourceMod Version**
```
    SourceMod Version: 1.9.0.6245
    SourcePawn Engine: 1.9.0.6245, jit-x86 (build 1.9.0.6245)
    SourcePawn API: v1 = 4, v2 = 12
    Compiled on: Jul 13 2018 05:05:50
    Built from: https://github.com/alliedmodders/sourcemod/commit/50b5bb1
    Build ID: 6245:50b5bb1
    http://www.sourcemod.net/
```
**Metamod Version:** 1.10 - build 966 - Linux

**sm plugins list**:
```
[SM] Listing 150 plugins:
  01 "csgotokens.com token changer" (1.5.0) by Adi - live4net
  02 "TecnoHard™ Admin Manager" (2.0_2017fix) by AbNeR @CSB
  03 "SM:RPG > Import THC RPG database" (1.0-533) by Peace-Maker
  04 "Player-Teleport by Dr. HyperKiLLeR" (1.2.0.0) by Dr. HyperKiLLeR
  05 "SM:RPG" (1.0-custom) by Jannik "Peace-Maker" Hartung, SeLfkiLL
  06 "Dice SM" (1.6.2) by Popoklopsi
  07 "Player Respawn" (1.6) by Rogue
  08 "OVERWATCH:Buttons" (1.0) by Potatoz
  09 "SourceTV / GOTV Protector" (1.0) by CrazyHackGUT aka Kruzya
  10 "AFK Manager" (4.2.5) by Rothgar
  11 "Fun Votes" (1.8.0.6036) by AlliedModders LLC
  12 "[CSS/CS:GO] AbNeR ResetScore" (1.5fix) by AbNeR_CSS
  13 "CS:GO Movement Unlocker" (1.0) by Peace-Maker
  14 "[ ZK Corporation ] - Block Commands" (1.0) by crashzk
  15 "SM Jailed Reasons" (v1.0) by Franc1sco steam: franug
  16 "Map Nominations" (1.8.0.6036) by AlliedModders LLC
  17 "Smart URL Remover" (1.6.2) by Totenfluch [Fix by Agent Wesker]
  18 "Basic Chat" (1.8.0.6036) by AlliedModders LLC
  19 "Paintball" (1.2.0) by otstrel.ru Team
  20 "SM:RPG > Effect Hub" (1.0-533) by Peace-Maker
  21 "SM Franug Valve Sprays" (1.5.3) by Franc1sco Steam: franug
  22 "[CS:GO/CSS] TecnoHard™ Clan Stats" (2.0) by MMYTH e AbNeR @CSB
  23 "MyJailbreak - Core" (Beta 13.0-0465720) by shanapu
  24 "[CSGO] Restart Round" (1.0) by NiGhT_CSGO & Edited By SaBaToN
  25 "SM:RPG > Command List" (1.0-533) by Jannik "Peace-Maker" Hartung
  26 "Basic Comm Control" (1.8.0.6036) by AlliedModders LLC
  27 "TecnoHard™ Freeday" (1.3) by AbNeR @CSB
  28 "Noblock for players and nades" (1.0) by tommie113
  29 "TecnoHard™ Lasers" (1.2_tecnohard2018) by AbNeR @CSB
  30 "SM:RPG > Counter-Strike Experience Module" (1.0-533) by Jannik "Peace-Maker" Hartung, SeLfkiLL
  31 "OVERWATCH:Chat" (1.0) by Potatoz
  32 "Simple Chat Processor (Redux)" (2.1.0) by Simple Plugins, Mini
  33 "VAC Status Checker" (2.2.0) by Stevo.TVR
  34 "Basic Votes" (1.8.0.6036) by AlliedModders LLC
  35 "RandomCycle" (1.8.0.6036) by AlliedModders LLC
  36 "Throwing Knives" (23-3-2016) by original by meng, Bacardi, Franc1sco franug
  37 "SM:RPG > Credit Gifting" (1.0-533) by Peace-Maker
  38 "SM:RPG Upgrade > Reduced Gravity" (1.0-533) by Jannik "Peace-Maker" Hartung
  39 "SM:RPG Upgrade > Adrenaline" (1.0-533) by Peace-Maker
  40 "SM:RPG Upgrade > Fire Grenade" (1.0-533) by Jannik "Peace-Maker" Hartung
  41 "SM:RPG Upgrade > Frost Pistol" (1.0-533) by Jannik "Peace-Maker" Hartung
  42 "SM:RPG Upgrade > Health regeneration" (1.0-533) by Jannik "Peace-Maker" Hartung
  43 "SM:RPG Upgrade > Medic" (1.0-533) by Jannik "Peace-Maker" Hartung
  44 "SM:RPG Upgrade > Increase Clipsize" (1.0-533) by Peace-Maker
  45 "SM:RPG Upgrade > Impulse" (1.0-533) by Jannik "Peace-Maker" Hartung
  46 "SM:RPG Upgrade > Firerate" (1.0-533) by Peace-Maker
  47 "SM:RPG Upgrade > Armor+" (1.0-533) by Jannik "Peace-Maker" Hartung
  48 "SM:RPG Upgrade > Ice Stab" (1.0-533) by Jannik "Peace-Maker" Hartung
  49 "SM:RPG Upgrade > Vampire" (1.0-533) by Jannik "Peace-Maker" Hartung
  50 "SM:RPG Upgrade > Reduced Fall Damage" (1.0-533) by Peace-Maker
  51 "SM:RPG Upgrade > Denial" (1.0-533) by Jannik "Peace-Maker" Hartung
  52 "SM:RPG Upgrade > Ice Grenade" (1.0-533) by Peace-Maker
  53 Disabled: "SM:RPG Upgrade > Poison Smoke" (1.0-533) by Jannik "Peace-Maker" Hartung
  54 "SM:RPG Upgrade > Fast Reload" (1.0-533) by Peace-Maker
  55 "SM:RPG Upgrade > Armor regeneration" (1.0-533) by Jannik "Peace-Maker" Hartung
  56 "SM:RPG Upgrade > Health+" (1.0-533) by Jannik "Peace-Maker" Hartung
  57 "SM:RPG Upgrade > Speed+" (1.0-533) by Jannik "Peace-Maker" Hartung
  58 "SM:RPG Upgrade > Antidote" (1.0-533) by Peace-Maker
  59 "SM:RPG Upgrade > Stealth" (1.0-533) by Jannik "Peace-Maker" Hartung
  60 "SM:RPG Upgrade > Bouncy Bullets" (1.0-533) by Peace-Maker
  61 "SM:RPG Upgrade > Resupply" (1.0-533) by Jannik "Peace-Maker" Hartung
  62 "SM:RPG Upgrade > Antiflash" (1.0-533) by Peace-Maker
  63 "SM:RPG Upgrade > Damage+" (1.0-533) by Jannik "Peace-Maker" Hartung
  64 "SM:RPG Upgrade > Fire Pistol" (1.0-533) by Jannik "Peace-Maker" Hartung
  65 Disabled: "SM:RPG Upgrade > Shrinking" (1.0-533) by Peace-Maker
  66 "SM:RPG Upgrade > Grenade Resupply" (1.0-533) by Jannik "Peace-Maker" Hartung
  67 "SM:RPG Upgrade > Armor Helmet" (1.0-533) by Peace-Maker
  68 "SM:RPG Upgrade > Long Jump" (1.0-533) by Jannik "Peace-Maker" Hartung
  69 "SM Valve Gloves" (3.0) by Franc1sco franug and hadesownage
  70 "[CS:GO] Fake Competitive Ranks/Coins/Profile" (1.5.2) by By: Laam4 - Edited: crashzk
  71 "Basic Info Triggers" (1.8.0.6036) by AlliedModders LLC
  72 "TecnoHard™ Motd" (1.1) by AbNeR @CSB
  73 "TecnoHard Chat Tags" (1.5.2) by AbNeR @CSB
  74 "Dynamic" (0.0.32) by Neuro Toxin
  75 "ArmsFix.smx"
  76 "OVERWATCH:Death" (1.0) by Potatoz
  77 "WebLync" (0.0.13) by Neuro Toxin
  78 "[CS:S/CS:GO] No Fall Damage" (1.0) by alexip121093 & Neoxx
  79 "[AWP] No-Scope Detector" (1.2.7) by Ak0 (Improved by Grey83 & onehand)
  80 "LR - Flying knife fight" (1.0) by KeidaS and Franc1sco franug
  81 "Last Request: Throwingknives battle" (1.0.1) by Franc1sco franug
  82 "SM Franug Hats" (3.3.2) by Franc1sco franug
  83 "SM Franug Hats" (3.3.2) by Franc1sco franug
  84 "Anti-Flood" (1.8.0.6036) by AlliedModders LLC
  85 "Player Commands" (1.8.0.6036) by AlliedModders LLC
  86 "SwapTeam" (1.2.6) by Rogue - Originally by MistaGee
  87 "[ ZK Corporation ] - Block Tag Grupos" (1.0) by crashzk
  88 "Team Names" (1.1) by Internet Bully
  89 "SM:RPG > Ban players" (1.0-custom) by Peace-Maker
  90 "CSGO Items" (1.5.0) by SM9
  91 "Ragdoll Modifications" (1.2.4) by Twisted|Panda
  92 "Fun Commands" (1.8.0.6036) by AlliedModders LLC
  93 "Music Kits" (1.0.0) by Bara
  94 "SM:RPG > Disable experience" (1.0-533) by Jannik "Peace-Maker" Hartung
  95 "[Any] Chat Logging Redux + RTF Support" (1.1.2) by McFlurry, Keith Warren (Drixevel)
  96 "REGEX word filter" (1.2) by Twilight Suzuka
  97 "Advanced Parachute" (1.0) by ESK0
  98 "Bad name kick / ban" (1.0.0) by 2NASTY4U
  99 "TecnoHard™ Xerife" (1.8fix2) by AbNeR @CSB
  100 "SM:RPG Upgrade > Poison Smoke" (1) by Jannik "Peace-Maker" Hartung
  101 "Server Advertisement" (2.7) by ESK0 & .#Zipcore
  102 "SM:RPG > Turbo Mode" (1.0-custom) by Jannik "Peace-Maker" Hartung
  103 "[ANY] Gifts" (2.3) by Zephyrus
  104 "Thirdperson | Mirrow Mode" (1.11) by Zephyrus
  105 "CSGOItems Item Log" (0.1) by SM9
  106 "Basic Commands" (1.8.0.6036) by AlliedModders LLC
  107 "OVERWATCH:Nades" (1.0) by Potatoz
  108 "Updater" (1.2.2) by GoD-Tony
  109 "RankMe" (3.0.3.Kento.29.1) by lok1, Scooby, Kento, pracc, Kxnrl, CrazyHackGUT
  110 "VoiceAnnounceEx" (2.1.2) by Franc1sco franug, Mini and GoD-Tony
  111 "MapChooser" (1.8.0.6036) by AlliedModders LLC
  112 "Skybox" (1.01) by Deathknife
  113 "TeamChange Unlimited" (1.2.2) by Sheepdude
  114 "Nextmap" (1.8.0.6036) by AlliedModders LLC
  115 "SM Access By Steam Group" (1.1) by Franc1sco franug
  116 "Rock The Vote" (1.8.0.6036) by AlliedModders LLC
  117 "[Any] Deluxe Godmode" (2.3.1) by DarthNinja
  118 "Sound Commands" (1.8.0.6036) by AlliedModders LLC
  119 "SM First Person View Models Interface" (3.1) by Franc1sco franug
  120 "Last-Request: LEADERBOARD" (1.0) by IT-KiLLER
  121 "DR.API SHOW DAMAGE" (1.1.5) by Dr. Api
  122 "Admin Help" (1.8.0.6036) by AlliedModders LLC
  123 "Admin File Reader" (1.8.0.6036) by AlliedModders LLC
  124 "SM FPVMI - Custom Weapons Menu" (ZK Servidores v3.1) by Franc1sco franug
  125 "SM:RPG > Reset interval" (1.0-533) by Jannik "Peace-Maker" Hartung
  126 "[CS:GO/CSS] AbNeR Round End Sounds" (3.6.2) by AbNeR_CSS
  127 "Client Preferences" (1.8.0.6036) by AlliedModders LLC
  128 "SM_Hosties v2" (2.3.0) by databomb & dataviruset & comando
  129 "Admin Menu" (1.8.0.6036) by AlliedModders LLC
  130 "TecnoHard™ Rendição" (1.2) by AbNeR @CSB
  131 "TecnoHard™ Plugins Block" (1.2) by AbNeR @CSB
  132 "DR.API HIGH PING KICKER" (1.0.2) by Dr. Api
  133 "SourceBans++: Main Plugin" (1.6.3_abner_2) by SourceBans Development Team, SourceBans++ Dev Team, AbNeR_CSS
  134 "SourceBans++: Admin Config Loader" (1.6.3) by AlliedModders LLC, SourceBans++ Dev Team
  135 "SourceBans++: Bans Checker" (1.6.3) by psychonic, Ca$h Munny, SourceBans++ Dev Team
  136 "SourceBans++: SourceSleuth" (1.6.3) by ecca, SourceBans++ Dev Team
  137 "SourceBans++ Report Plugin" (1.6.3) by RumbleFrog, SourceBans++ Dev Team
  138 "Limit Ban Duration" (1.0.2_abner_2) by Twisted|Panda, AbNeR_CSS
  139 "SourceBans++ Discord Plugin" (1.0.4) by RumbleFrog, SourceBans++ Dev Team
  140 "SourceBans++: SourceComms" (1.6.3) by Alex, SourceBans++ Dev Team
  141 "CT Ban - Max Bans" (1.0.0) by databomb
  142 "CT Ban" (2.0.3) by databomb
  143 "Give Weapon" (2.0) by Author: Kiske - Edited: crashzk & biielzk
  144 "SM Franug CS:GO Stat Trak" (ZK Servidores v1.4.1) by Franc1sco franug
  145 "SM Franug CS:GO NameTag" (ZK Servidores v2.0) by Franc1sco franug
  146 "CS:GO GiveNamedItem Hook Franug Edition" (6.1 private version) by Franc1sco franug and Neuro Toxin
  147 "SM CS:GO Weapon Paints" (ZK Servidores v6.9) by Franc1sco franug
  148 "SM CS:GO Franug Knives" (3.1 private version) by Franc1sco franug
  149 "fix_knives.smx"
  150 "SM Wear option for Weapon Paints" (1.0.1 private version) by Franc1sco franug
```
Log Erros: 
[errors_20181018.log](https://github.com/shanapu/MyJailbreak/files/2493197/errors_20181018.log)

Config `csgo/addons/sourcemod/configs/MyJailbreak`: 
[MyJailbreak.zip](https://github.com/shanapu/MyJailbreak/files/2493211/MyJailbreak.zip)

Config `/csgo/cfg/MyJailbreak`: [MyJailbreak 2.zip](https://github.com/shanapu/MyJailbreak/files/2493208/MyJailbreak.2.zip)

**Comment by paulocrash** [comment]
Sorry, it was my mistake, in case I downloaded the newer version, I just did not send it to the server... :)

I'll send and I'll come back with the results!

About the MyJailBreak plugins, I do not use them all in the case, I use only `myjailbreak.smx`, the rest is all disabled.

I thought the `myjailbreak.smx` itself that mutated the prisoners in the case... I'll be seeing **Plugin Hosties** better.

Thanks for now, I will be making these changes and return.

**Comment by shanapu** [comment]
myjailbreak.smx alone doesn't make much. It's a core plugin, a interface for the all other myjb plugins and the eventdays.
The only features of myjailbreak.smx when you run it alone are:

command: sm_endround - end the current round
convar: sm_myjb_shoot_buttons - allow to shot buttons instead of pressing use.

all other functions a depending on other myjb plugins.

**Comment by shanapu** [comment]
> Sorry, it was my mistake, in case I downloaded the newer version, I just did not send it to the server... :)
> I'll send and I'll come back with the results!

I think this can be closed. 
If error still appears, please open new issue
